From dc721b362a2a46ae25b93f05fdec4abf72202bf4 Mon Sep 17 00:00:00 2001
From: Liang Qi <liang.qi@qt.io>
Date: Tue, 6 Jul 2021 12:00:17 +0200
Subject: [PATCH 025/165] opengl: fix a typo in
 QOpenGLPaintDevice::dotsPerMeterY()

Fixes: QTBUG-95005
Pick-to: 6.2 6.1 5.15 5.12
Change-Id: I88ab9b757a5cfee520e07ac6280e31f9211342f0
Reviewed-by: Laszlo Agocs <laszlo.agocs@qt.io>
(cherry picked from commit 85831bcfe6ce36740c524d641ada79c65aadd89b)
---
 src/gui/opengl/qopenglpaintdevice.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gui/opengl/qopenglpaintdevice.cpp b/src/gui/opengl/qopenglpaintdevice.cpp
index 3920a10467..9e7d2861fd 100644
--- a/src/gui/opengl/qopenglpaintdevice.cpp
+++ b/src/gui/opengl/qopenglpaintdevice.cpp
@@ -331,7 +331,7 @@ void QOpenGLPaintDevice::setDotsPerMeterX(qreal dpmx)
 
 void QOpenGLPaintDevice::setDotsPerMeterY(qreal dpmy)
 {
-    d_ptr->dpmx = dpmy;
+    d_ptr->dpmy = dpmy;
 }
 
 /*!
-- 
2.36.1

